### PR TITLE
minor enhancements

### PR DESCRIPTION
**Pull request by trishankatdatadog**
* replace wget w/ curl, which we know does TLS verification
* simplify CLI invocation
* replace rootname filename, which starts w/ 0 now (but will replaced w/ 1)

### CODE REVIEW

File: scripts/verify.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ if [[ ! -z "$1" ]]; then
     git checkout VERIFY
 fi
 
-./verify --root piv-attestation-ca.pem --repository $REPO --tuf-root $REPO/repository/1.root.json
+./verify --root piv-attestation-ca.pem --repository $REPO --tuf-root $REPO/repository/0.root.json
```
Comment (asraa) [inline scripts/verify.sh]: should we change this to 1 since the version 1 code is already merged?

**Review by asraa — COMMENTED** [review]
Yay thank you for testing this out!!!

**Comment by trishankatdatadog** [comment]
> Yay thank you for testing this out!!!

No problem 🙂 Is this good?

**Review by asraa — APPROVED** [review]
Yep! LGTM

**Pull request merged**